backend/src/main/java/com/javahelp/backend/endpoint/APIGatewayResponse.java

```diff
@@ -18,10 +18,18 @@
  */
 public class APIGatewayResponse {
 
+    /**
+     * Response code for unauthorized requests
+     */
+    public static int FORBIDDEN = 403;
+
+    /**
+     * Response code for unauthenticated requests
+     */
+    public static int UNAUTHENTICATED = 401;
+
     /**
      * Response code for unsupported media type errors
-     * <p>
-     * Should be returned by NonJSONBodyResponse
      */
     public static int UNSUPPORTED_MEDIA_TYPE = 415;
```

Comment:
Should we set these variables to final since the status code are set and constant?